### ISSUE TITLE
Register "to", "with", or "against"

### ISSUE THREAD

**Issue by cwickert** [SUSE/doc-styleguide#232]
Our docs are using different wordings for the product registration: Sometimes you register _with_ the Suse Customer Center (SCC), sometimes _to_ it and sometimes _against_ it. "with" is most common but means both the server (SCC, Suse Manager, customer's RMT server, or a SUSE registration server in a public cloud) as well as the tool used for registration (`yast regisration`, `SUSEConnect` and `registercloudguest`).

We should codify this in the terminology table along with instructions what is what.

Native speakers, what do you prefer?

**Comment by tahliar** [comment]
Ahhh, this is a topic I've struggled with both here and at my previous job. I can say definitively that I don't like "against". However, I see that I've used "with" in the Liberty docs, but used "to" in a Red Hat Satellite doc I worked on some years ago... but used "with" in a Red Hat Virtualization doc I worked on before that. Oh dear.

Now that I'm actively thinking about it, I think Christoph makes a good point that "with" can also mean "using", whereas "to" is less ambiguous. If, just for example, you included both the server and the tool in the sentence, you could say "Register **to** the RMT server **with** SUSEConnect" (sure, it would probably be better to say "Use SUSEConnect to register to/with the RMT server", but let's just go with it for the sake of comparison), whereas "Register **with** the RMT server **with** SUSEConnect" is pretty clunky. You could also say "Register **with** the RMT server **using** SUSEConnect", I guess. 

**TL;DR**: TBH I'm not really certain about "register to" vs "register with", but Christoph brings up a logical argument against "with". 

Let's see what Padraig thinks, too.

**Comment by PadraigDillon1** [comment]
This is a tricky one. Fristly, I agree with Christoph and Tahlia that 'with' sounds a little clunky. 

I can see use cases for all of the examples listed by Tahlia. However, for me the most natrual sounding example would be this one  - "Use SUSEConnect to register to/with the RMT server",

As for an entry in the Terminology Database for this. The database might not be the correct place for this as it's not necessarily a term. It's possible more a style guide issue.

**Comment by cwickert** [comment]
To make things even more confusing, we have statements like "register to get updates" or "users may only register with a valid account and password," so register _to_ and _with_ will always be confusing. That leaves _against_.

A brief look at the doc-sle repo shows that "register with" is the most common, however when talking about SCC, SuMa, SMT, or RMT, then "register against"is most common.

**Comment by dariavladykina** [comment]
I've tried to look more closely at the usage in `doc-sle` and reportedly, `with` is the leader with 24 occurrences, followed by `against` = 5 and `by using` = 5. `Register to` and `using` each enjoy just 3. 
The sweeping majority is sth like `Registration with SCC/RMT`, e.g. *Registering and deregistering machines with SUSEConnect.*
Moreover, we also have `register at [the server]`, `on [the server] `and even `via`.

Probably we could give a recommendation and examples for sentences including "server + tool"?
Something like:
Use "to" when writing about registering to a server, e.g. *This is required on the client machines registered to the RMT server.* Use "with" when writing about the tool used for registration, e.g. *Create the new domain and register it with libvirt.*

Feel free to update the instruction and examples!

**Comment by dariavladykina** [comment]
Closing this as it was merged.
